src/api/stats/getAmmPrices.js

```diff
@@ -179,6 +180,7 @@ const pools = [
   ...thugsPools,
   ...cakeLpV1Pools,
   ...cakeLpPools,
+  ...waultPools,
```

Comment:
let's add it on top the list, it's read from bottom to top and cakeLPs are used as first oracle source.